README.md

```diff
@@ -49,3 +49,14 @@ If you want to run your app locally in production mode, run `mvn spring-boot:run
 For documentation on using Vaadin Flow and Spring, visit [vaadin.com/docs](https://vaadin.com/docs/flow/spring/tutorial-spring-basic.html).
 
 For more information on Vaadin Flow, visit [vaadin.com/flow](https://vaadin.com/flow).
+
+## Data base
+The project uses postgres 11.5. [Documentation](https://www.postgresql.org/docs/11/index.html)
+
+Currently for local development we use [postgres docker image](https://hub.docker.com/_/postgres)
+
+To run docker use the following command:
+
+*  on linux:
+`docker run -p 5432:5432 -e POSTGRES_DB=brn -e POSTGRES_PASSWORD=$PG_PASSWORD -e POSTGRES_USER=$PG_USER postgres:11`
+* on windows: `?`
```

Comment:
pls someone test that this branch works fine on windows(I use linux) and update that line in readme(as far as I remember _docker run_ command a little bit different on win and linux, but I'm not sure about that)